### ISSUE TITLE
package is not executable

### ISSUE THREAD

**Issue by SimplyKnownAsG** [mgedmin/restview#52]
The package cannot be executed using `python -m restview .`. 
```
No module named restview.__main__; 'restview' is a package and cannot be directly executed
```

NOTE: One can get around this using `python -m restview.restviewhttp`.

It would be nice to not have to specify the the `.restviewhttp` particularly since there is only one entry point into the program (as near as I can tell). This can be done with something like.

```python
# src/restview/__main__.py
from restview import restviewhttp
restviewhttp.main()
```

This type of functionality can be useful with multiple Pythons, and also on windows, where the scripts\*.exe files may not be in the path, be restricted for silly IT reasons, etc.

**Comment by mgedmin** [comment]
This is an excellent idea!

**Comment by mgedmin** [comment]
Fixed in 2.9.0, released to PyPI today.
